chat_stream_test.go

```diff
@@ -0,0 +1,43 @@
+package gogpt
+
+import (
+	"context"
+	"errors"
+	"github.com/sashabaranov/go-gpt3/internal/test"
+	"io"
+	"testing"
+)
+
+func TestClient_CreateChatCompletionStream(t *testing.T) {
+	config := DefaultConfig(test.GetTestToken())
```

Comment:
Without setting the BaseURL, the test would not contact the Mock server. See the previous stream test: https://github.com/sashabaranov/go-gpt3/blob/58d99eb220f747e9712f28995d08465f77e025f3/stream_test.go#L43

Let's not bother with a mock server here and have a test for a real API. To obtain a token, please use logic similar to [here](https://github.com/sashabaranov/go-gpt3/blob/58d99eb220f747e9712f28995d08465f77e025f3/api_test.go#L14)

```go
apiToken := os.Getenv("OPENAI_TOKEN")
if apiToken == "" {
	t.Skip("Skipping testing against production OpenAI API. Set OPENAI_TOKEN environment variable to enable it.")
}
```

Reply:
> Without setting the BaseURL, the test would not contact the Mock server. See the previous stream test:
> 
> https://github.com/sashabaranov/go-gpt3/blob/58d99eb220f747e9712f28995d08465f77e025f3/stream_test.go#L43
> 
> Let's not bother with a mock server here and have a test for a real API. To obtain a token, please use logic similar to [here](https://github.com/sashabaranov/go-gpt3/blob/58d99eb220f747e9712f28995d08465f77e025f3/api_test.go#L14)
> 
> ```go
> apiToken := os.Getenv("OPENAI_TOKEN")
> if apiToken == "" {
> 	t.Skip("Skipping testing against production OpenAI API. Set OPENAI_TOKEN environment variable to enable it.")
> }
> ```

Sorry, I don't know much about this testing process yet. In order not to cause you any additional trouble, I temporarily removed it.